feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.ts

```diff
@@ -136,4 +136,15 @@ export class ConfiguratorAttributeHeaderComponent
     const images = this.attribute.images;
     return images && this.hasImage ? images[0] : undefined;
   }
+
```

Comment:
function `isAttributeGroup` and `getConflictMessageKey` have `groupType` as parameter. This parameter can be removed, because `groupType` is input variable in the component. It can be used directly.
```
isAttributeGroup(): boolean {
   return Configurator.GroupType.ATTRIBUTE_GROUP === this.groupType; 
}
```
You can even remove this function `isAttributeGroup` and do the check in html.

Reply:
I removed `groupType` as parameter from both methods but still prefer to call `isAttributeGroup` method in HTML. Otherwise, I think is will not readable anymore.

Reply:
But this is another breaking change. Shouldn't we focus on composition of the components with this PR, and deal with isolated issues from components later? Because I fear that it will get too big, and that it will be hard to cover breaking changes correctly. @WeizhengSap what do you think?

Reply:
See #13286